runtime/interpreter/test-framework.go

```diff
@@ -22,22 +22,51 @@
 package interpreter
 
 import (
+	"github.com/onflow/cadence/runtime/common"
 	"github.com/onflow/cadence/runtime/errors"
+	"github.com/onflow/cadence/runtime/sema"
 )
 
 // TestFramework is the interface to be implemented by the test providers.
 // Cadence standard library talk to the test providers via this interface.
 // This is used as a way to inject test provider dependencies dynamically.
 //
 type TestFramework interface {
-	RunScript(code string, args []Value) ScriptResult
+	RunScript(code string, args []Value) *ScriptResult
+
+	CreateAccount() (*Account, error)
+
+	AddTransaction(
+		code string,
+		authorizers []common.Address,
+		signers []*Account,
+		args []Value,
```

Comment:
(sorry for being nitpicky about style)

```suggestion
	RunScript(code string, arguments []Value) *ScriptResult

	CreateAccount() (*Account, error)

	AddTransaction(
		code string,
		authorizers []common.Address,
		signers []*Account,
		arguments []Value,
```